test/es6/objlit.js

```diff
@@ -75,36 +75,28 @@ var tests = [
             var obj = {
                 ["foo" + "bar"]         : 1,
                 [1 * 10 * 10]           : 2,
-                ["foobar", "notfoobar"] : 3, // Should evaluate to notfoobar

```

Comment:
To ensure we're parsing the right starting grammar production we should have a test that uses one of the production rules that are unique to that production.  Simplest would be a simple assignment, so something like
`[a = 'baz']: 3,`.

Here and the other cases below.